tests/test-war-smoke/src/main/docker/Dockerfile

```diff
@@ -0,0 +1,2 @@
+FROM ${jetty.test.image}
+ADD root /app
```

Comment:
I don't think that really explains it.
In any case, I was able to get the app to run with `ADD root /app`, but I still get a directory listing if I open `localhost:8080/` rather than `localhost:8080/hello`. So, there's still something not right with this approach. I really think we should switch to something like `ADD test-war-smoke-0.1.0-SNAPSHOT.war $JETTY_BASE/webapps/root.war`, and create a separate issue to make sure we have support for building images with exploded WAR.